test/runtests.jl

```diff
@@ -27,85 +27,85 @@ function MosekOptimizerWithFallback()
 end
 
 
-@testset "SolverName" begin
```

Comment:
Were these changes intentional?

Reply:
Yes, this is WIP, I still have a few tests to exclude or fix